추가기능요구사항.md

```
@@ -0,0 +1,256 @@
+        - 수정 대상 객체의 id 파라미터, 수정할 값 파라미터
+- `delete`
+- [x] 관련된 도메인도 같이 삭제합니다.
+  BinaryContent(프로필), UserStatus
+
+---
+
+### 의존성
+
+- 같은 레이어 간 의존성 주입은 순환 참조 방지를 위해 지양합니다. 다른 Service 대신 필요한 Repository 의존성을 주입해보세요.
+
+![](https://res.cloudinary.com/dhabktrg9/image/upload/v1753954803/lzuibpjly8bpgi7ldssl.png)
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)